discord/ext/commands/converter.py

```diff
@@ -159,6 +159,80 @@ class UserConverter(IDConverter):
 
         return result
 
+class MessageConverter(IDConverter):
+    """Converts to a :class:`discord.Message`.
+
+    The lookup strategu is as follows (in order):
```

Comment:
I feel like this lookup strategy is not optimal. It should be reordered to 1, 3, 2. 

Also URI is actually URL.

Reply:
(also, "strategu" should be "strategy")